src/cljs/rems/cart.cljs

```diff
@@ -62,8 +62,9 @@
   [:tr.cart-item
    [:td.title (get-localized-title item language)]
    [:td.commands
-    [remove-from-cart-button item language]
-    (when apply-button? [apply-button [item] language])]])
+    [:div.commands.justify-content-end
```

Comment:
Default is already the end?

Reply:
Default for a table is actually start, so for the cart and catalogue we must override it.